edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,74 @@
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4056, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4086,materialTypeName: TestMaterial1, itemBarcode: 2222}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4089, itemId: 388830d5-95db-4528-95b6-4ec9d37d4056 }
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 188830d5-95db-4528-95b6-4ec9d34d4086, itemId: 388830d5-95db-4528-95b6-4ec9d37d4086 }
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 1111,username:  testUser1}
+    * def userId = createUserResponse.createUserRequest.id
+
+    Given path 'patron/account/' + userId
+    When method GET
+    Then status 200
+    And match response.totalCharges.amount == amount+amount
+
+  Scenario: Return loans for a patron
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 2222,username: testUser2}
+    * def userId = createUserResponse.createUserRequest.id
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostPolicies')
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4057, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4090, materialTypeName: TestMaterial2, itemBarcode: 3333}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostCheckOut') { itemBarcode: 3333,userBarcode: 2222 }
+
+    Given path 'patron/account/' + userId
+    When method GET
+    Then status 200
+    And match response.totalLoans == 1
+
+  Scenario: Return fees/fines per item for a patron
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 3333,username:  testUser3}
+    * def userId = createUserResponse.createUserRequest.id
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4058, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4091,materialTypeName: TestMaterial3, itemBarcode: 4444}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine') {barcode: 3333,materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4091, itemId: 388830d5-95db-4528-95b6-4ec9d37d4058 }
+
+    Given path 'patron/account/' + userId + '?includeCharges=true'
+    When method GET
+    Then status 200
+    And match response.totalChargesCount == 1
```

Comment:
Could it be worth checking that it is the expected fee / fine, e.g. with the ID or barcode?

Reply:
yes, i will add itemId.